server/index.js

```diff
@@ -0,0 +1,11 @@
+var express = require('express');
+var bodyParser = require('body-parser');
+var db = require('../database/database.js');
+var app = express();
+const port = process.env.PORT || 3002;
+
+app.get('/restaurantInfo/:id', bodyParser.json(), db.findRestaurant);
```

Comment:
this ":id" params is cool. I didnt know you can do that. Will have to add it into my code.